tests/preprocessing/test_triplegs.py

```diff
@@ -146,6 +147,26 @@ def test_generate_trips_dtype_consistent(self):
         assert stps["next_trip_id"].dtype == "Int64"
         assert tpls["trip_id"].dtype == "Int64"
 
+    def test_compare_to_old_trip_function(self):
```

Comment:
would be nice to run this function on Geolife. I can do that after the other comments are answered